grafana-stack/nginx.yaml

```diff
@@ -40,6 +40,7 @@ spec:
     targetPort: nginx-port
   selector:
     app: nginx
+  loadBalancerIP: 104.197.200.129
```

Comment:
Is this the best way to specify the load balancer ip? Seems kind of specific and odd.